tasks/update-defaults-mode.rake

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+# updates the defaults mode configuration
+task 'update-defaults-mode', [:defaults_file] do |t, args|
+
+  defaults_file = args[:defaults_file]
+  if defaults_file.nil? || defaults_file.empty?
+    raise ArgumentError, 'Missing required defaults_file, please provide one as a rake argument.'
+  end
+
+  puts "Loading defaults from: #{defaults_file}"
+  defaults = JSON.load(File.read(defaults_file))
+
+  default_mode_docs = ["<p>The following `:default_mode` values are supported: </p>"]
```

Comment:
Should these hardcoded strings live in the actual documentation, and maybe just code generate multiple dynamic sections like all of the modes and configuration maps?

Reply:
Yeah - I think that makes sense.

Reply:
NVM - this doesn't work as we need all of the links to be processed together by kramdown.